src/js/pages/jobs/JobTaskDetailPage.js

```diff
@@ -20,19 +26,25 @@ class JobTaskDetailPage extends React.Component {
       { label: "Logs", routePath: routePrefix + "/logs" }
     ];
 
+    const job = MetronomeStore.getJob(id);
     const task = MesosStateStore.getTaskFromTaskID(taskID);
 
-    let breadcrumbs;
+    let breadcrumbs = (
+      <JobsBreadcrumbs tree={DCOSStore.jobTree} item={job} details={false} />
```

Comment:
Yeah, this is a  cheap _workaround_ to align the functionality with the old implementation. I'm planning to completely redo the breadcrumbs (remove the type specific components and introduce utils to generate the list of crumbs), but went for a less intrusive refactoring as this change is urgently needed to eventually fix DCOS_OSS-1033. I will file a JIRA to properly refactor the breadcrumbs and leave this as is fi that's ok with you.